packages/tls-liquidsoap/tls-liquidsoap.1/opam

```diff
@@ -32,9 +33,9 @@ build: [
 dev-repo: "git+https://github.com/savonet/liquidsoap.git"
 url {
   src:
-    "https://github.com/savonet/liquidsoap/archive/refs/tags/v2.2.0.tar.gz"
+    "https://github.com/savonet/liquidsoap-release-assets/releases/download/v2.3.0/liquidsoap-2.3.0.tar.gz"
```

Comment:
This is changing an existing package, maybe a mistake?

Reply:
Reverted!